test/integration/helpers/html-utils.ts

```diff
@@ -52,10 +52,18 @@ export class HtmlElementLookup {
     expectSvgElementClassAttribute(svgGroupElement, 'bpmn-task');
   }
 
-  expectServiceTask(bpmnId: string, additionalClasses?: string[]): void {
+  expectServiceTask(bpmnId: string, options?: { additionalClasses?: string[]; overlayLabel?: string }): void {
     const svgGroupElement = this.findSvgElement(bpmnId);
     expectSvgTask(svgGroupElement);
-    expectSvgElementClassAttribute(svgGroupElement, HtmlElementLookup.computeClassValue('bpmn-service-task', additionalClasses));
+    expectSvgElementClassAttribute(svgGroupElement, HtmlElementLookup.computeClassValue('bpmn-service-task', options?.additionalClasses));
+
+    const overlayGroupElement = document.querySelector<SVGGElement>(`#${this.bpmnVisualization.graph.container.id} > svg > g > g:nth-child(3) > g[data-bpmn-id="${bpmnId}"]`);
```

Comment:
ℹ️ I am working on a refactoring PR that moves the query selectors along with other selectors used in tests. Both PR are changing the code, so merge conflicts are going to happen.